compiler/src/dotty/tools/dotc/printing/Formatting.scala

```diff
@@ -67,10 +70,10 @@ object Formatting {
         def show(x: TypeComparer.ApproxState) = TypeComparer.ApproxState.Repr.show(x)
 
       given Show[Showable]                            = ShowAny
-      given Show[Shown]                               = ShowAny
```

Comment:
Why was that dropped?

Reply:
I think either it was hiding something or I was concerned it was being used.  Then it was no longer needed so I kept it out.  But I'm fine with keeping it in.